packages/grafana-ui/src/components/DataTable/types.ts

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from 'react';
+import { CellProps, DefaultSortTypes, IdType, SortByFn } from 'react-table';
+
+export interface Column<TableData extends object> {
+  /**
+   * ID of the column.
+   * Set this to the matching object key of your data or `undefined` if the column doesn't have any associated data with it.
```

Comment:
Should remove the mention of `undefined` from here

Reply:
🤦 of course i forgot this